sources/packages/backend/apps/workers/src/controllers/disbursement/_tests_/e2e/disbursement.controller.saveDisbursementSchedules.e2e-spec.ts

```diff
@@ -1,78 +1,50 @@
-import { Test, TestingModule } from "@nestjs/testing";
 import {
-  Application,
   ApplicationStatus,
   AssessmentTriggerType,
   DisbursementOveraward,
   DisbursementOverawardOriginType,
   DisbursementSchedule,
   DisbursementScheduleStatus,
   DisbursementValueType,
-  EducationProgramOffering,
-  Student,
-  StudentAssessment,
-  User,
 } from "@sims/sims-db";
 import {
+  createE2EDataSources,
   createFakeApplication,
   createFakeDisbursementOveraward,
   createFakeEducationProgramOffering,
   createFakeStudent,
   createFakeStudentAssessment,
   createFakeUser,
+  E2EDataSources,
 } from "@sims/test-utils";
 import { createFakeDisbursementSchedule } from "@sims/test-utils/factories/disbursement-schedule";
 import { createFakeDisbursementValue } from "@sims/test-utils/factories/disbursement-value";
-import { DataSource, Repository } from "typeorm";
-import { ZBClient } from "zeebe-node";
-import { WorkersModule } from "../../../workers.module";
 import { createFakeSaveDisbursementSchedulesPayload } from "./save-disbursement-schedules-payloads";
-import { DisbursementController } from "../disbursement.controller";
+import { DisbursementController } from "../../disbursement.controller";
 import {
   FAKE_WORKER_JOB_RESULT_PROPERTY,
   MockedZeebeJobResult,
-} from "../../../../test/utils/worker-job-mock";
+} from "../../../../../test/utils/worker-job-mock";
+import { createTestingAppModule } from "../../../../testHelpers";
 
-describe("Disbursement Schedule Service - Create disbursement", () => {
-  let userRepo: Repository<User>;
```

Comment:
Really great simplification. This one serves as an example of how the use of the `E2EDataSources` can make the E2E code cleaner 😉